### PR TITLE
rebalance beverage quench based on oral hydration index

### PR DESCRIPTION
**Pull request by XygenSS**
#### Summary
Balance "rebalance beverage quench based on oral hydration index"

#### Purpose of change

Beverage hydration, as far as I can tell, was chosen arbitrarily at the time of addition. This PR attempts to match it with real-world data.

#### Describe the solution

Using clean water as a reference (1.00/50 quench), apply the following values:

1. 1.00/50 quench
- Water
- Soda and sparkling water
- Sports and energy drinks
- Coffee and tea, including those made with milk
- Beer

2. 1.30/65 quench
- All fruit juices
- Apple cider, cranberry juice, OJ, lemonade and other -ades
- Tomato juice, V8, Virgin mary
- Crispy cranberry

3. 1.40/70 quench
- Milk, curdled milk, soy milk, almond milk, sweet milk, sweet fortified milk

4. 1.50/75 quench
- Hydration drink

5. Unaffected
- Drinks containing chocolate
- Eggnog and spiked eggnog
- Pina colada, Aryan
- Wine
- Distilled alcoholic drinks

This PR also adjusts condensed milk's quench value to 1 (was 12).

#### Describe alternatives you've considered
- Bumping coffee and tea beverages containing milk higher than 50
- Bumping chocolate drinks to 50
- Bumping eggnog to 50

#### Testing
JSON change, game loads without error

#### Additional context
1-s2 0-S000291652206556X-gr2_lrg

All data is based on https://www.sciencedirect.com/science/article/pii/S000291652206556X
PDF download and web viewing available free of charge.

### CODE REVIEW

**Comment by Venera3** [comment]
Only the ORS / milk / orange juic

**Comment by Venera3** [comment]
God dammit, sorry.

**Comment by XygenSS** [comment]
it's still a fact that most of the affected beverages are way less hydrating than they should be, you think I should simply make all of them 50 quench except for juices, ORS and milks?

**Comment by Venera3** [comment]
Yes, normalizing the rest to 50 should work.
As long as we have a single value for both subjective and objective thirst I'd go with representing the latter, where it's not that different (based on this study, etc).

**Comment by SirPendrak** [comment]
If you are going to touch dairy, please change condensated milk. It has 20 charges in 500ml, each has 12 quench. Thats 240 total for half litre. I have no data, but it should probably be closer to -1 / +1 quench per charge with all that sugar

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "comestible_type": "DRINK",
     "symbol": "~",
     "quench": 22,
-    "calories": 50,
+    "calories": 70,
```
Comment (GuardianDll) [inline data/json/items/comestibles/drink.json:38]: also here, github suggestion is stoopid and do not allow me to suggest a fix

**Comment by GuardianDll** [comment]
are you gonna fix `drink_other.json`?

**Comment by Maddremor** [comment]
This is not a change request as such, but I have brought it up before so I might out it down on the record:
If we want to keep the quench values from veering off something reasonable in the future, we could introduce new values to JSON such that hydration is the mass of the comestible, times the fraction of it that is water, multiplied by a hydration modifier that accounts for both positive effects from things like electrolytes and dehydration from excess salt/sugar/whatever. This way you wouldn't need to change stuff if portion sizes were changed, and it's hard to just eyeball a wildly wrong number.

**Comment by XygenSS** [comment]
> are you gonna fix `drink_other.json`?

I don't see anything that needs fixing in that file, should I?

**Comment by GuardianDll** [comment]
just saw there is no changes in it, if it's entirely correct, then it's fine

**Comment by Maleclypse** [comment]
Squashed this so it wouldn't have one of its commits in the record be "I am a massive idiot".

**Pull request merged**